docs/source/setup_docker.rst

```diff
@@ -119,7 +119,11 @@ While the containers are running (ie after running the docker-compose up command
 
 .. code-block:: bash
 
-    docker exec -it seed_web_1 python manage.py test --config.settings.docker_dev
+    docker exec -it seed_web_1 python manage.py test --settings config.settings.docker_dev
+
+Add the setting  :code:`--nocapture` in order to see :code:`stdout` while running tests.  You will need to do this in order to make use of debugging as described below or the output to your debug commands will not display until after the break point has passed and the tests are finished.
```

Comment:
woah didn't think about it intercepting debugger statements but that makes sense. good fix 👍